blocks/api/test/parser.js

```diff
@@ -99,15 +106,21 @@ describe( 'block parser', () => {
 		} );
 
 		it( 'should create the requested block with no attributes if it exists', () => {
-			registerBlock( 'core/test-block', {} );
+			registerBlock( 'core/test-block', {
+				edit: edit,
```

Comment:
Minor: We could take advantage of [ES2015 object property shorthand](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#New_notations_in_ECMAScript_2015) to simplify this:

```js
registerBlock( 'core/test-block', { edit, save } );
```